src/openrct2-ui/windows/Sign.cpp

```diff
@@ -482,44 +505,48 @@ rct_window * window_sign_small_open(rct_windownumber number){
  *
  *  rct2: 0x6E6145
  */
-static void window_sign_small_mouseup(rct_window *w, rct_widgetindex widgetIndex)
+static void window_sign_small_mouseup(rct_window * w, rct_widgetindex widgetIndex)
 {
-    rct_banner* banner = &gBanners[w->number];
-    sint32 x = banner->x << 5;
-    sint32 y = banner->y << 5;
+    rct_banner * banner = &gBanners[w->number];
+    sint32       x      = banner->x << 5;
+    sint32       y      = banner->y << 5;
```

Comment:
Do we really have to do this? It looks so weird.

Reply:
I _wouldn't mind_ to have this code style rule changed. Depends a bit on what the rest of team thinks, but I think that @IntelOrca and @duncanspumpkin might agree with you here.

Reply:
I'm with @marijnvdwerf, its way too much. I don't mind this on data tables but not necessarily everywhere.

Reply:
And even weirder when it aligns the assignment in a for loop with a variable declaration. I thought this rule was only applied to stuff outside functions - like fields in a class - I didn't expect it inside function bodies.